codeboarding_web/static/app.css

```
@@ -0,0 +1,335 @@
+
+.detail-empty.hidden { display: none; }
+
+.detail-content { display: flex; flex-direction: column; gap: .5rem; }
+.detail-content.hidden { display: none; }
+
+#detail-name {
+  margin: 0;
+  font-size: 1rem;
+  font-weight: 700;
+  color: #E6E6E6;
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)